src/styles/header.scss

```diff
@@ -0,0 +1,95 @@
+@font-face {
+  font-family: Roboto;
+  font-weight: 500;
+  src: url(https://fonts.google.com/specimen/Roboto);
+}
+
+html {
+  font-family: Roboto, Arial, sans-serif;
+  font-weight: 500;
+}
+
+body {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+.header {
+  display: flex;
+  padding: 0 50px;
+  background-color: #fff;
+  align-items: center;
+  justify-content: space-between;
+}
+
+.logo {
+  height: 40px;
+  width: 40px;
+  display: block;
+}
+
+.logo_image {
+  height: 100%;
+  width: 100%;
+}
+
+.nav_list {
+  position: relative;
+  text-decoration: none;
```

Comment:
The 'text-decoration' property is not typically applied to a list element like '.nav_list'. Consider removing it or applying it to a more appropriate element, such as a link.